### DISCUSSION TITLE
when I config accelerate config to deepspeed nvme path, the path does not pass to deepspeed.py

### DISCUSSION THREAD

**Discussion by mobilejammer** [bghira/SimpleTuner#1256] [General]
my accelerate config as follow:
compute_environment: LOCAL_MACHINE
debug: false
deepspeed_config:
  deepspeed_moe_layer_cls_names: ''
  gradient_accumulation_steps: 1
  gradient_clipping: 1.0
  offload_optimizer_device: nvme
  offload_optimizer_nvme_path: /data/deepspeed_offload
  offload_param_device: nvme
  offload_param_nvme_path: /data/deepspeed_offload
  zero3_init_flag: true
  zero3_save_16bit_model: true
  zero_stage: 3
distributed_type: DEEPSPEED
downcast_bf16: 'no'
dynamo_config:
  dynamo_backend: INDUCTOR
enable_cpu_affinity: false
machine_rank: 0
main_training_function: main
mixed_precision: bf16
num_machines: 1
num_processes: 8
rdzv_backend: static
same_network: true
tpu_env: []
tpu_use_cluster: false
tpu_use_sudo: false
use_cpu: false
![1735556751470](https://github.com/user-attachments/assets/47d1fe6b-18a8-4255-973e-a1d69376b096)
![1735556804582](https://github.com/user-attachments/assets/9fcd4a8d-fc93-46d1-b12b-325f9bc58ee8)

**Comment by bghira** [comment]
mate, see the OPTIONS.md document for offload location setting.

for some reason, Accelerate doesn't provide this value. it's likely a bug in Accelerate.
